core/pfe-core/controllers/color-context.scss

```diff
@@ -16,7 +22,25 @@
   --pfe-broadcasted--link-decoration--visited: var(--pfe-theme--link-decoration--visited--on-dark, none);
 }
 
-:host([on=saturated]) {
```

Comment:
I'm not sure if this a side effect fix or was intentional but before `:host([on="light|dark|saturated|..."]`  would override `--pfe--broadcasted--text` even when set as a custom CSS value on the parent `<pfe-band>` of a `<pfe-accordion-header>`  

Screenshot from current production [pfe-accordion/demo](https://patternflyelements.com/components/accordion/demo/):
![image](https://user-images.githubusercontent.com/13666/162762560-987363a6-128e-4956-8b3a-de4f75a89cf8.png)

[Deploy preview ](https://deploy-preview-1976--patternfly-elements.netlify.app/components/accordion/demo/)for this PR
![image](https://user-images.githubusercontent.com/13666/162762688-db4f95aa-3f81-43e2-9663-085722864a16.png)

Setting the value `--pfe-broadcasted-text` now seems to cascade correctly 🎉.

Reply:
:fire: looks like a win to me